### DISCUSSION TITLE
Crossfade of two video sources

### DISCUSSION THREAD

**Discussion by Adi-18** [bytedeco/javacv#1593] [Q&A] [answered]
Is it possible to combine two videos, lets say 20% of video 1 over 80% of video 2?

I've been looking for a way for cross fading.

**Comment by saudet** [comment, marked as answer]
There appears to be a filter for this, yes: https://ffmpeg.org/ffmpeg-filters.html#xfade

**Reply by Adi-18** [reply]
Thank you for the Informationen, it looks intresting.
My only problem is, I am Java developer and have no idea how to implement it there.

**Reply by saudet** [reply]
We need support for multiple inputs, so something like shown here in this unit test should work:
https://github.com/bytedeco/javacv/blob/master/platform/src/test/java/org/bytedeco/javacv/FrameFilterTest.java#L168

**Reply by Adi-18** [reply]
Thank you. I'll give a try

**Reply by fackee** [reply]
> Thank you. I'll give a try

Hello,The xfade filter has worked for you?

**Reply by Adi-18** [reply]
Hi. Thank you. Now I found time to check this.
As I see, I now have a split screen. VID1 left, VID2 right (its a part of wipe fade).

What I am looking for is having VID2 **over** VID1 with dynamic transparency lets say 70%.
Later it should transform transparency from 0% to 100% depend of input value.
